src/obr/core/core.py

```diff
@@ -191,3 +196,30 @@ def unlink(fn):
             unlink(fn)
     else:
         unlink(fns)
+
+
+def map_view_folder_to_job_id(view_folder: str) -> dict[str, str]:
+    """Creates a mapping from the view schema to the original jobid
+
+    Returns:
+    ========
+        A dictionary with jobid: view_folder
+    """
+    ret = {}
+    base = Path(view_folder)
+    if not base.exists():
+        return {}
+    for root, folder, file in os.walk(view_folder):
+        for i, fold in enumerate(folder):
+            path = Path(root) / fold
+            job_id = None
+            if path.is_symlink():
+                job_id = path.resolve().name
+                # only keep path parts relative to the start of
+                # of the view folder
+                rel_fold = [
+                    p for p in path.absolute().parts if p not in base.absolute().parts
+                ]
+                ret[job_id] = "/".join(rel_fold)
```

Comment:
I have a last remark for this. 
You basically want to get the relative path from `path` to `base`, correct?
Wouldn't this just be
 ```suggestion
                if path.absolute().is_relative_to(base.absolute()):
                    ret[job_id] = str(path.absolute().relative_to(base.absolute())
```
?